targon/internal/targon/callbacks.go

```diff
@@ -438,6 +475,7 @@ func getWeights(c *Core) ([]types.U16, []types.U16, error) {
 				}
 			}
 		}
+		minerScores[uid] = thisScore
```

Comment:
this is happening before normalization, so it doesn't corelate to actual payout.